packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/createDummyChangeListItems.with..st

```diff
@@ -0,0 +1,21 @@
+running
+createDummyChangeListItems: anInteger with: aBoolean
+
+	| dummyChangeRecords changeListItems |
+	changeListItems := OrderedCollection new: anInteger.
+	dummyChangeRecords := (OrderedCollection new: 2)
+		add: self createChangeRecordMethod;
+		add: self createChangeRecordClassComment;
+		yourself.
+	
+	1 to: anInteger do: [:each |
+		| index record |
+		index := each \\\ dummyChangeRecords size + 1.
```

Comment:
What do the backslashes do?

Reply:
It's Smalltalk syntax for Modulo, because a modulo: method or % operator would be too easy